scripts/converters/to-markdown.js

```diff
@@ -1,10 +1,13 @@
 const path = require('path');
 const TurndownService = require('turndown');
+const HTMLtoJSX = require('htmltojsx');
 
 const getCategories = require('../utils/get-categories');
 const getFrontmatter = require('../frontmatter/get-frontmatter');
 const { TYPES, BASE_DIR } = require('../constants');
 
+const htmlToJSXConverter = new HTMLtoJSX({ createClass: false });
```

Comment:
having trouble understanding the documentation for this site, what does setting createClass to false do here?

Reply:
This is the only html -> jsx converter I could find. Unfortunately its super old and still uses the `React.createClass` API (it was the primary API before JS classes were mainstream). Here we just want the raw JSX and don't want it wrapped in a component via `React.createClass`. Disabling this ensures we get the raw JSX output back.